### PR TITLE
Update versions for tarantool modules

### LINKED ISSUE #208: Updated expirationd to 1.1.0
Updated expirationd to version 1.1.0

### LINKED ISSUE #215: Update module versions in docker files
We need to update module versions to newer ones in all docker files. Versions for the following modules should be updated and verified:

- avro-schema
- expirationd
- queue
- connpool
- shard
- http
- curl
- pg
- mysql
- memcached
- metrics
- prometheus
- mqtt
- gis
- gperftools

### PR DESCRIPTION
**Pull request by ylobankov**
This patch updates the version for each tarantool module to the latest
tag of the module where it's possible.

Also, a build issue was fixed for Centos 7. It was impossible to
download the `pgdg-redhat-repo-latest.noarch.rpm` package by the link
from the docker file. So the link was updated.

Closes #208 
Closes #215

### CODE REVIEW

File: dockerfiles/alpine_3.5 [hunk 1]
```diff
@@ -18,17 +18,17 @@ ENV TARANTOOL_VERSION=${TNT_VER} \
...
     LUAROCK_CONNPOOL_VERSION=1.1.1 \
-    LUAROCK_HTTP_VERSION=1.1.0 \
+    LUAROCK_HTTP_VERSION=1.2.0 \
     LUAROCK_MEMCACHED_VERSION=1.0.1 \
-    LUAROCK_METRICS_VERSION=0.10.0 \
+    LUAROCK_METRICS_VERSION=0.12.0 \
     LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
     LUAROCK_TARANTOOL_MYSQL_VERSION=2.0.1 \
-    LUAROCK_TARANTOOL_MQTT_VERSION=1.2.1 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.0 \
```
Comment (Totktonada) [inline dockerfiles/alpine_3.5:31]: AFAIU, we will not tag new images based on Apline 3.5.

(Sure, we should revisit the repository layout in a future. Possibly use some template engine.)

**Review by Totktonada — APPROVED** [review]
LGTM. Looks quite straightforward.

I would like to see a summary of your testing results.

I don't know, whether there is any sense to update `dockerfiles/alpine_3.5` if there will be no images that will be build with the updated code. It looks rather confusing than consistent. It is up to you.

File: dockerfiles/alpine_3.5 (hunk 1 above)
Comment (ylobankov) [reply dockerfiles/alpine_3.5]: Ok, removed changes for Alpine 3.5.

**Comment by ylobankov** [comment]
> LGTM. Looks quite straightforward.
> 
> I would like to see a summary of your testing results.
> 
> I don't know, whether there is any sense to update `dockerfiles/alpine_3.5` if there will be no images that will be build with the updated code. It looks rather confusing than consistent. It is up to you.

* `TAG=1 OS=centos DIST=7 VER=1.x PORT=5200 make -f .gitlab.mk build` - works
* `TAG=2 OS=centos DIST=7 VER=2.x PORT=5200 make -f .gitlab.mk build` - works
* `TAG=1 OS=alpine DIST=3.9 VER=1.x PORT=5200 make -f .gitlab.mk build` - doesn't work, but looks like it is not related to my patch
```
+ : '---------- luarocks ----------'
+ : ldoc
+ tarantoolctl rocks install ldoc
Warning: Failed searching manifest: Failed fetching manifest for http://rocks.tarantool.org/ - Failed downloading http://rocks.tarantool.org/manifest - /root/.cache/luarocks/http___rocks.tarantool.org_/manifest
Warning: Failed searching manifest: Failed fetching manifest for http://luarocks.org/repositories/rocks - Failed downloading http://luarocks.org/repositories/rocks/manifest - /root/.cache/luarocks/http___luarocks.org_repositories_rocks/manifest
Warning: Failed searching manifest: Failed fetching manifest for http://rocks.tarantool.org/ - Failed downloading http://rocks.tarantool.org/manifest - /root/.cache/luarocks/http___rocks.tarantool.org_/manifest
Warning: Failed searching manifest: Failed fetching manifest for http://luarocks.org/repositories/rocks - Failed downloading http://luarocks.org/repositories/rocks/manifest - /root/.cache/luarocks/http___luarocks.org_repositories_rocks/manifest
Warning: Failed searching manifest: Failed fetching manifest for http://rocks.tarantool.org/ - Failed downloading http://rocks.tarantool.org/manifest - /root/.cache/luarocks/http___rocks.tarantool.org_/manifest
Warning: Failed searching manifest: Failed fetching manifest for http://luarocks.org/repositories/rocks - Failed downloading http://luarocks.org/repositories/rocks/manifest - /root/.cache/luarocks/http___luarocks.org_repositories_rocks/manifest

Error: No results matching query were found.
The command '/bin/sh -c set -x     && apk add --no-cache --virtual .run-deps.2         mariadb-connector-c-dev         libpq         cyrus-sasl         mosquitto-libs         libev     && apk add --no-cache --virtual .build-deps.2         git         cmake         make         coreutils         gcc         g++         postgresql-dev         lua-dev         musl-dev         cyrus-sasl-dev         mosquitto-dev         libev-dev         wget         unzip     && mkdir -p /rocks     && : "---------- proj (for gis module) ----------"     && wget -O proj.tar.gz http://download.osgeo.org/proj/proj-4.9.3.tar.gz     && mkdir -p /usr/src/proj     && tar -xzf proj.tar.gz -C /usr/src/proj --strip-components=1     && (cd /usr/src/proj;         ./configure;         make -j ${NPROC};         make install)     && rm -r /usr/src/proj     && rm -rf /usr/src/proj     && rm -rf /proj.tar.gz     && : "---------- geos (for gis module) ----------"     && wget -O geos.tar.bz2 http://download.osgeo.org/geos/geos-3.6.0.tar.bz2     && mkdir -p /usr/src/geos     && tar -xjf geos.tar.bz2 -C /usr/src/geos --strip-components=1     && (cd /usr/src/geos;         ./configure;         make -j ${NPROC};         make install)     && rm -r /usr/src/geos     && rm -rf /usr/src/geos     && rm -rf /geos.tar.bz2     && : "---------- luarocks ----------"     && : "ldoc"     && ${ROCKS_INSTALLER} install ldoc     && : "lua-term"     && ${ROCKS_INSTALLER} install lua-term     && : "avro"     && ${ROCKS_INSTALLER} install avro-schema $LUAROCK_AVRO_SCHEMA_VERSION     && : "expirationd"     && ${ROCKS_INSTALLER} install expirationd $LUAROCK_EXPERATIOND_VERSION     && : "queue"     && ${ROCKS_INSTALLER} install queue $LUAROCK_QUEUE_VERSION     && : "connpool"     && ${ROCKS_INSTALLER} install connpool $LUAROCK_CONNPOOL_VERSION     && : "vshard"     && ${ROCKS_INSTALLER} install vshard $LUAROCK_VSHARD_VERSION     && : "http"     && ${ROCKS_INSTALLER} install http $LUAROCK_HTTP_VERSION     && : "pg"     && ${ROCKS_INSTALLER} install pg $LUAROCK_TARANTOOL_PG_VERSION     && : "mysql"     && ${ROCKS_INSTALLER} install mysql $LUAROCK_TARANTOOL_MYSQL_VERSION     && : "memcached"     && ${ROCKS_INSTALLER} install memcached $LUAROCK_MEMCACHED_VERSION     && : "metrics"     && ${ROCKS_INSTALLER} install metrics $LUAROCK_METRICS_VERSION     && : "prometheus"     && ${ROCKS_INSTALLER} install prometheus $LUAROCK_TARANTOOL_PROMETHEUS_VERSION     && : "mqtt"     && ${ROCKS_INSTALLER} install mqtt $LUAROCK_TARANTOOL_MQTT_VERSION     && : "gis"     && ${ROCKS_INSTALLER} install gis $LUAROCK_TARANTOOL_GIS_VERSION     && : "gperftools"     && ${ROCKS_INSTALLER} install gperftools $LUAROCK_TARANTOOL_GPERFTOOLS_VERSION     && : "---------- remove build deps ----------"     && apk del .build-deps.2' returned a non-zero code: 1
make: *** [build] Error 1
```
* `TAG=2 OS=centos DIST=3.9 VER=2.x PORT=5200 make -f .gitlab.mk build` - works

**Comment by Totktonada** [comment]
It seems, you closes #208 as well.

Let's triage ldoc problem.

**Comment by ylobankov** [comment]
> It seems, you closes #208 as well.
> 
> Let's triage ldoc problem.

Filed #217.

**Pull request merged**